Libraries/Control/Control.h

```diff
@@ -46,11 +47,9 @@ class Control {
     void turnLED();
     // Turn on a LED for 1 second.
     void blinkLED();
-    // Initialze the two leds;
-    void initializeLED();
 
     const uint8_t kLimitSupDegrees = 35;
-    const uint8_t kLimitInfDegrees = 4;
+    const uint8_t kLimitInfDegrees = -11;
```

Comment:
Does this actually work? look that it's an unsigned constant and you're giving it a -11.